### PR TITLE
Fix resize in Firefox on Android

### PR DESCRIPTION
**Pull request by samhed**
This fixes issues #446 and #463 by using a different approach of getting the full size of the canvas. Instead of using window.innerWidth and window.innerHeight this would resize the canvas to fit the noVNC_container which covers the entire page.

Edit: This also fixes issue #465

### CODE REVIEW

**Comment by samhed** [comment]
This also fixes issue #465

**Comment by DirectXMan12** [comment]
In general, looks good.  I'll try and test it soon.

**Comment by samhed** [comment]
> Once #451 merges, you can just use getPosition here -- it will returns the width and height (or you could just use getBoundingClientRect directly).

You mean instead of parseInt from window.getComputedStyle? Like this

    var h = Util.getPosition(container).height

?

**Comment by DirectXMan12** [comment]
> You mean instead of parseInt from window.getComputedStyle? Like this

>     var h = Util.getPosition(container).height

> ?

Yep!

**Comment by samhed** [comment]
This PR is now updated to reflect the changes @DirectXMan12  made in #451

**Comment by samhed** [comment]
I'm getting a bit limited in my time to work on this soon, basically I only have today and tomorrow available. Would it be possible for you @DirectXMan12 to verify that I haven't broken the scaling etc. and come with potential comments today or tomorrow?

I have tested scaling and as far as I can tell I haven't broken anything.

File: include/ui.js [hunk 1]
```diff
@@ -258,6 +218,55 @@ var UI;
...
+        },
+
+        getCanvasLimit: function () {
+            var container = $D('noVNC_container');
+
+            // Hide the scrollbars until the size is calculated
+            container.style.overflow = "hidden";
+
+            var w = Util.getPosition(container).width;
+            var h = Util.getPosition(container).height;
```
Comment (DirectXMan12) [inline include/ui.js]: This could be more effecient (`var pos = Util.getPosition(container);`, etc), but it's not a big deal.

**Comment by DirectXMan12** [comment]
LGTM! :+1:  Seems to work for me as well.

**Comment by samhed** [comment]
Thank you for taking the time @DirectXMan12 

> This could be more effecient (var pos = Util.getPosition(container);, etc), but it's not a big deal.

Good catch, I fixed it in 16b3ef7

**Pull request merged**